test/community/label_propagation.jl

```diff
@@ -4,6 +4,7 @@
     for g in testgraphs(g10)
         z = copy(g)
         for k = 2:5
+            Random.seed!(0)
```

Comment:
revert this. It's a known issue but forcing a seed in an unrelated piece of code is not good for this PR.

Reply:
Ok; apologies for going off topic.

Reply:
No worries. If you'd like to make a PR that fixes this, please do so; we'd want to loop in @jpfairbanks and others on the review to see whether this approach (fixed seed) makes sense or whether there's a better way to handle failures in random functions.